backgroundScript.js

```diff
@@ -1,6 +1,6 @@
 chrome.runtime.onInstalled.addListener(() => {
-	const environment = "prod";
-	// const environment = "dev";
+	// const environment = "prod";
+	const environment = "dev";
```

Comment:
Please revert this. You can refrain from committing this file while you are testing the extension locally.

Reply:
Please revert this.